libzhlgen/lua/parsefuncs.lua

```diff
@@ -479,15 +481,15 @@ for k,fd in pairs(tfiles) do
         if func.struct then
             -- Generic code and extra dependencies for structs
             local s = structs[func:cname()]
-            
             if s then
                 if func.generic_code then
                     s.generic_code = (s.generic_code or "")..func.generic_code
                 end
-                
+
                 if func.depends then
                     for _,d in ipairs(func.depends) do
                         local dname = d:cname()
+						if (string.find(dname, func:cname().."::")) then overridenChild[dname] = true end
```

Comment:
Incorrect whitespace mess 🥇

Reply:
Turns out the entire parsefuncs.lua file is made of a mix of 4 space and Tab whitespace mess. I fixed them all for a 4 space indent.